packages/projects-docs/pages/learn/getting-started/setting-up-ios.mdx

```diff
@@ -9,37 +9,45 @@ description: With this guide you'll learn how to get started with the iOS app
 
-For the full CodeSandbox experience, we recommend linking your GitHub account. This will allow you to import your GitHub repositories and contribute to them faster.
+> 💡 For the full CodeSandbox experience, we recommend linking your GitHub account. This will allow you to import your GitHub repositories and contribute to them faster.
 
 ### Select the Appropriate Workspace
 
-When you log in, your personal workspace will be selected as the default. If you're part of a team, you can easily switch to your team's workspace by selecting it from the sidebar. On an iPhone, you can access the screen to switch workspaces by tapping the **Back** button in the screen's top left corner.
+When you log in, your personal workspace will be selected as the default. If you're part of a team, you can easily switch to your team's workspace by selecting it from the sidebar.
+
+On an iPhone, you can access the screen to switch workspaces by tapping the **Back** button in the screen's top left corner.
+
+> 💡 [Create a team](https://codesandbox.io/dashboard?create_team=true) to fully experience how CodeSandbox can [accelerate your team's workflow](https://codesandbox.io/develop-full-stack-apps).
 
 ![Screenshot of the iOS Dashboard](/ios-dashboard.png)
 
 ### Accessing Sandboxes and Repositories
 
-If you are an existing CodeSandbox user, you will be presented with a list of sandboxes and repositories you have recently interacted with under the **Recent** section after selecting a workspace. To view all available sandboxes and repositories, navigate through the different categories in the sidebar or use the **Search** functionality available at the top of the screen.
+If you are an existing CodeSandbox user, you will see a list of sandboxes and repositories you have recently interacted with under the **Recent** section after selecting a workspace.
+
+To view all available sandboxes and repositories, navigate through the different categories in the sidebar, such as **Repositories** and **Sandboxes** or use the **Search** functionality, which is available at the top of the screen.
 
 ### Creating a New Sandbox or Repository
 
-You also have the option of starting from scratch by creating a new sandbox or importing a repository from GitHub.
+You can start coding by using one of the existing CodeSandbox templates or by importing a repository from GitHub.
 
-#### Importing a Repository from GitHub
+#### Creating a Sandbox from a Template
 
-To import a repository from GitHub, simply tap on the **New Repository** button to start the import process. You'll be prompted to link your CodeSandbox account with a GitHub account if you haven't already.
+Start coding in seconds by using one of the official [CodeSandbox templates](https://codesandbox.io/s/). It's a quick and easy way to get started with any technology without any setup required.
```

Comment:
Same as before, I would avoid linking to parts of the web client. Consider that the user might not be authenticated into CodeSandbox in the browser as well.